scripts/release/mule/Makefile.mule

```diff
@@ -46,32 +46,30 @@ ci-build: ci-setup buildsrc gen
 		 NO_BUILD=True VARIATIONS=$(OS_TYPE)/$(ARCH) \
 		 scripts/build_packages.sh $(OS_TYPE)/$(ARCH)
 
-package-$(OS_TYPE)-$(ARCH): package-deb-$(OS_TYPE)-$(ARCH) package-rpm-$(OS_TYPE)-$(ARCH)
-
-package-deb-$(OS_TYPE)-$(ARCH):
+package-deb:
 	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/deb/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
 
-package-rpm-$(OS_TYPE)-$(ARCH):
-	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/rpm/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
-
-package-docker-$(OS_TYPE)-$(ARCH):
+package-docker:
 	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/docker/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
 
-package-test-$(OS_TYPE)-$(ARCH): package-test-deb-$(OS_TYPE)-$(ARCH) package-test-rpm-$(OS_TYPE)-$(ARCH)
+package-rpm:
+	scripts/release/mule/package/$(OS_TYPE)/$(ARCH)/rpm/package.sh $(OS_TYPE) $(ARCH) $$WORKDIR
```

Comment:
Why are the scripts specific to os type and arch, and also take those in as arguments?

Reply:
I thought about hardcoding the OS_TYPE and ARCH in the shell scripts, but for now I'd like to leave that as generic as possible for possible re-use.

Reply:
rather remove the os_type and arch from the path?

Reply:
We're going to want the flexibility of building packages for different operating systems and architectures.

Reply:
Could we make one package script that generically packages all builds in the tmp/node_pkg folder? The os/arch information is in the path to each bin folder under that directory so this information could be captured there right?

Reply:
To minimize the amount of rewriting necessary, we could have a package script that references these os/arch specific scripts for each folder in the tmp/node_pkg dir. This could be better for us since then we can use one jenkins agent to do all of our package, rather than using extra compute to spin up an agent for each os/arch type